shared/constants/teams.js

```diff
@@ -327,6 +327,11 @@ const getDisabledReasonsForRolePicker = (
   teamname: Types.Teamname,
   memberToModify: ?string
 ): Types.DisabledReasonsForRolePicker => {
+  const canManageMembers = getCanPerform(state, teamname).manageMembers
+  if (canManageMembers) {
+    // If you're an implicit admin, the tests below will fail for you, but you can still change roles.
+    return {}
```

Comment:
If this is a subteam, `"Subteams cannot have owners."`

Reply:
Guess this applies to the rest of this function too. Want to split into a separate ticket?